Examples/Tests/collider_relevant_diags/analysis_multiple_particles.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+
+import numpy as np
+import openpmd_api as io
+import pandas as pd
+from scipy.constants import c, e, hbar, m_e
+
+sys.path.append('../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+sys.path.append('../../../../warpx/Tools/Parser/')
+from input_file_parser import parse_input_file
+
+E_crit = m_e**2*c**3/(e*hbar)
+B_crit = m_e**2*c**2/(e*hbar)
+
+def chi(ux, uy, uz, Ex, Ey, Ez, Bx, By, Bz):
+    gamma = np.sqrt(1.+ux**2+uy**2+uz**2)
+    vx = ux / gamma * c
+    vy = uy / gamma * c
+    vz = uz / gamma * c
+    tmp1x = Ex + vy*Bz - vz*By
+    tmp1y = Ey - vx*Bz + vz*Bx
+    tmp1z = Ez + vx*By - vy*Bx
+    tmp2 = (Ex*vx + Ey*vy + Ez*vz)/c
+    chi = gamma/E_crit*np.sqrt(tmp1x**2+tmp1y**2+tmp1z**2 - tmp2**2)
+    return chi
+
+def dL_dt():
+    series = io.Series("diags/diag2/openpmd_%T.h5",io.Access.read_only)
+    iterations = np.asarray(series.iterations)
+    lumi = []
+    for n,ts in enumerate(iterations):
+        it = series.iterations[ts]
+        rho1 = it.meshes["rho_beam_e"]
+        dV = np.prod(rho1.grid_spacing)
+        rho1 = it.meshes["rho_beam_e"][io.Mesh_Record_Component.SCALAR].load_chunk()
+        rho2 = it.meshes["rho_beam_p"][io.Mesh_Record_Component.SCALAR].load_chunk()
+        q1 = np.unique(it.particles["beam_e"]["charge"][io.Mesh_Record_Component.SCALAR].load_chunk())
+        q2 = np.unique(it.particles["beam_p"]["charge"][io.Mesh_Record_Component.SCALAR].load_chunk())
```

Comment:
Are you using `np.unique` here because all the electrons (positrons) have the same charge? If it is the case I would suggest  checking this explicitly.

Reply:
Thanks @lucafedeli88, is commit [f207759](https://github.com/ECP-WarpX/WarpX/pull/4024/commits/f207759e082902dac199a331a0a9323d9fb844b2) accomplishing what you had in mind?